### ISSUE TITLE
Using the validation set

### ISSUE THREAD

**Issue by DanielMallia** [DAA233/generative-inpainting-pytorch#53]
Since the validation set section is currently commented out, how do you think it is best used for this case of neural network? 

Thanks

**Comment by daa233** [comment]
I used the ImageNet validation set but I'm not sure if it's best.
